docs/pages/divider.js

```diff
@@ -1,64 +1,388 @@
+    </Flex>
+  );
+}
 `}
-      />
-    </MainSection.Subsection>
-  </MainSection>,
-);
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Create your own divider with a different color or stroke style."
+            defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading accessibilityLevel="none" size="sm">{title}</Heading>
+      {
+        text.map((item, idx) => <Text key={idx} size="md" underline><Link href="#" target="blank">{item}</Link></Text> )
+      }
+    </Flex>
+  );
 
-export default function DividerPage(): Node {
-  return <CardPage cards={cards} page="Divider" />;
+  return (
+    <Flex direction="column" gap={10} width={300}>
+      <Block title="Get started" text={['Add a Pinterest widget', 'Upgrade the Pinterest App', 'Interact with Idea Pins']} />
+      <hr style={{  borderTop: '3px dashed blue' }}/>
+      <Block title="Manage Account" text={['Edit notification settings', 'Two-factor authentication', 'Log in and out of Pinterest']} />
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use horizontal Dividers from edge to edge."
+            defaultCode={`
+function Example() {
+  const Block = ({ title, url, locked }) => (
+    <Flex gap={2} alignItems="center">
+      <Box maxWidth={60}>
+        <Mask rounding={4}>
+          <img alt="example image" src={url} style={{ maxWidth: '100%', display: 'block' }} />
+        </Mask>
+      </Box>
+      <Flex.Item flex="grow">
+        <Text weight="bold" size="md">
+          {title}
+        </Text>
+      </Flex.Item>
+      {locked ? (
+        <Icon accessibilityLabel="" icon="lock" size={12} />
+      ) : (
+        <AvatarGroup
+          accessibilityLabel="Collaborators."
+          size="sm"
+          collaborators={[
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ]}
+        />
+      )}
+    </Flex>
+  );
+
+  return (
+    <Flex direction="column" gap={10} width={300}>
+      <Block title="Home decor" url="https://i.ibb.co/121JJzC/stock7.jpg" locked/>
+      <Divider />
+      <Block title="Plants" url="https://i.ibb.co/FY2MKr5/stock6.jpg" />
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Inset the Divider in a way that causes it to be free-floating or separated from content."
```

Comment:
nit: no article needed, just "Inset Divider in a way…"

Reply:
DONE